app/src/main/java/com/raduitache/myanimelist/auth/impl/AuthNavRouteImpl.kt

```diff
@@ -0,0 +1,236 @@
+import androidx.compose.foundation.layout.IntrinsicSize
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.heightIn
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.widthIn
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material3.Button
+import androidx.compose.material3.CircularProgressIndicator
+import androidx.compose.material3.ElevatedButton
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextField
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.AnnotatedString
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.input.OffsetMapping
+import androidx.compose.ui.text.input.TransformedText
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.max
+import androidx.compose.ui.viewinterop.AndroidView
+import androidx.hilt.navigation.compose.hiltViewModel
+import com.raduitache.myanimelist.BuildConfig
+import com.raduitache.myanimelist.R
+import com.raduitache.myanimelist.auth.AuthNavRoute
+import kotlinx.coroutines.delay
+import javax.inject.Inject
+import kotlin.math.min
+
+class AuthNavRouteImpl @Inject constructor() : AuthNavRoute("auth/auth-screen", emptyList()) {
+
+    @Composable
+    override fun Content() {
+        val authViewModel: AuthViewModel = hiltViewModel()
+        val state = authViewModel.authViewState.collectAsState().value
+
+        var email by remember {
+            mutableStateOf("")
+        }
+
+        var password by remember {
+            mutableStateOf("")
+        }
+
+        var username by remember {
+            mutableStateOf("")
+        }
+
+        LazyColumn(
+            modifier = Modifier
+                .fillMaxSize()
+                .padding(16.dp),
+            horizontalAlignment = Alignment.CenterHorizontally
+        ) {
+            item {
+                Box(
+                    modifier = Modifier
+                        .size(width = 128.dp, height = 64.dp)
+                        .background(Color.Red)
+                )
+                Box(modifier = Modifier.heightIn(min = 120.dp))
```

Comment:
What's the point of these?

Reply:
LASA-MA VERE IN PACE CAT TIMP MERGE NU DAM IN PRODUCTIE